packages/viewer/src/utils.spec.ts

```diff
@@ -0,0 +1,176 @@
+import { JSDOM } from "jsdom";
+import { serializeHtmlToXml } from "./utils"; // adjust the import path as needed
+
+global.DOMParser = new JSDOM().window.DOMParser;
+global.XMLSerializer = new JSDOM().window.XMLSerializer;
+global.document = new JSDOM().window.document;
```

Comment:
¿Por qué se hace esto? Veo que DOMParser de la línea anterior se usa luego, pero ni XMLSerializer ni document se referencian en ningún sitio en esta batería de pruebas.

Reply:
Vale, veo que son necesarios porque internamente la función que se está probando hace uso de XMLSerializer y de document. Pondría un comentario reflejando precisamente eso.

Reply:
Perfecto, añado el comentario. 
De todos modos no quedo muy contento con cómo se construyen los tests. Debería de ser configurable con nx sin tener que hacer tantos malabares. 
Lo intenté pero no salió a la primera y cómo se aleja bastante del alcance del issue original quizás es buena idea tenerlo en cuenta como punto de mejora posterior (con un TO-DO en esa misma parte).

Reply:
No me preocuparía por eso en exceso Ubaldo, seamos más prácticos en esta ocasión, con lo que hemos conseguido yo estoy más que contento.